### PR TITLE
implements a custom typescript config possibility in compilers/typesc…

### PR DESCRIPTION
**Pull request by webforgeeu**
…ript-compiler.js

If a tsconfig.jest.json is found it will load that one instead of the default tsconfig.json

### CODE REVIEW

File: lib/compilers/typescript-compiler.js [hunk 1]
```diff
@@ -34,7 +35,12 @@ function getTypescriptConfig () {
   if (cachedConfig) {
     return cachedConfig
   } else {
-    const { config } = tsconfig.loadSync(process.cwd())
+    const tscPath = process.cwd()
+    let tscFile = 'tsconfig.jest.json'
+    if (!fs.existsSync(tscPath+ '/'+ tscFile)){
+        tscFile = false
+    }
+    const { config } = tsconfig.loadSync(tscPath, tscFile)
```
Comment (eddyerburgh) [inline lib/compilers/typescript-compiler.js]: We should use a jest global to get the file name, similarly to how ts-jest does it—https://github.com/kulshekhar/ts-jest/blob/master/src/preprocessor.ts#L17

You can pass a global in in the Jest config object:
```js
{
  "jest": {
    "globals": {
      "vue-jest": {
        "tsConfigFile": "my-tsconfig.json"
      }
    }
  }
}
```

File: README.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
-# vue-jest
+# vue-jest-kn
```
[the remark this answers is not in the document]
Comment (webforgeeu) [reply README.md]: had to change it temporarily to have an npm with it to integrate.
Will be changed back once I have the time to refactor to jest-globals.
Should have made another fork .. my bad ;)

**Comment by dcherman** [comment]
@webforgeeu Are you still interested in working on this?  I need this as well, but don't want to step on your toes if you're actively working on it / completing it soon.

**Comment by webforgeeu** [comment]
@dcherman 
Feel free to take that on.
My workload is a bit too high at the moment as I'd afford to spend time on that.
I'd be more than happy to help out if you need some support here and there, but I'm afraid I can't complete that in the next weeks or so.

**Pull request closed without merging**